Missing changes from main

sathish-progress opened the pull request:
<!--- Provide a short summary of your changes in the Title above -->
Get `inspec-6` branch prepared to be merged to `main`.

## Description
<!--- Describe your changes in detail, what problems does it solve? -->
I tried pulling in changes from `main` to `inspec-6` branch so that we have updated changes from `main` to resolve conflicts that happen [here](https://github.com/inspec/inspec/compare/inspec-6). Once we merge this changes to `inspec-6` branch we are basically in sync with the `main` branch without conflicts.
Also instead of pulling in changes from `main` for waiver file reader, Keeping Nikitha's refactoring that was done on `inspec-6` and just adding some bug fix for waiver 

## Related Issue
<!--- If you are suggesting a new feature or change, please create an issue first -->
<!--- Please link to the issue, discourse, or stackoverflow here: -->

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [X] Breaking change (a content change which would break existing functionality or processes)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] I have read the **CONTRIBUTING** document.

tnir committed:
Sync up the default branch as main (#6239)

Signed-off-by: Takuya Noguchi <takninnovationresearch@gmail.com>

Signed-off-by: Takuya Noguchi <takninnovationresearch@gmail.com>

chef-ci committed:
Bump version to 5.20.2 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

marijus-ravickas committed:
resolves inspec/inspec#6042 (#6043)

Signed-off-by: Marijus Ravickas <marijus.ravickas@gmail.com>

Signed-off-by: Marijus Ravickas <marijus.ravickas@gmail.com>

ahasunos committed:
Enhance `lxc` resource to test properties (#6243)

* ENHANCE: Extend lxc resource to test the properties

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* TEST: Extend test for introduced properties

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* DOCS: Extend test for introduced properties

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* FIX: Correct exception class to handle bad yaml

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* Doc Review

Signed-off-by: Deepa Kumaraswamy <dkumaras@progress.com>

* FIX: Handle specific exception to handle bad yaml

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

Signed-off-by: Sonu Saha <sonu.saha@progress.com>
Signed-off-by: Deepa Kumaraswamy <dkumaras@progress.com>
Co-authored-by: Deepa Kumaraswamy <dkumaras@progress.com>

chef-ci committed:
Bump version to 5.20.3 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
CFINSPEC-462: Fixes inspec sign breaks when there is period or dot in the profile name (#6261)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.20.4 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Nik08 committed:
CFINSPEC-374 DSL keyword `only_applicable_if` added (#6229)

* Not applicable if logic addition

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Changes from not applicable if to only applicable if

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Fix to enable placing only_applicable_if at any position in control and for keeping impact zero intact

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Doc change added for only_applicable_if

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Doc Review

Signed-off-by: Deepa Kumaraswamy <dkumaras@progress.com>

* Added generic examples that supports cross platform for testing

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Yet another build fix due to changes in test

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>
Signed-off-by: Deepa Kumaraswamy <dkumaras@progress.com>
Co-authored-by: Deepa Kumaraswamy <dkumaras@progress.com>

chef-ci committed:
Bump version to 5.21.0 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Nik08 committed:
CFINSPEC-458 Oracle DB session resource compatibility with AIX-C shell (#6257)

* Oracle DB session resource compatibility with AIX-Cshell

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Fixes in oracledb session unit test using oracle_query_string

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

chef-ci committed:
Bump version to 5.21.1 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

tnir committed:
Replace faraday_middleware with faraday-follow_redirects (#6238)

faraday_middleware was already DEPRECATED in 2022-01.
Only FaradayMiddleware::FollowRedirects from faraday_middleware has been
used, and so we can replace it with faraday-follow_redirects.

faraday-follow_redirects 0.3.0 requires faraday ~> 1.0, which is as same
as "faraday_middleware 1.0.0 or higher.

Signed-off-by: Takuya Noguchi <takninnovationresearch@gmail.com>

Signed-off-by: Takuya Noguchi <takninnovationresearch@gmail.com>

chef-ci committed:
Bump version to 5.21.2 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
Update Faraday gem (#6267)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.3 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

dependabot[bot] committed:
Bump test-kitchen from 3.3.1 to 3.3.2 in /omnibus (#6227)

Bumps [test-kitchen](https://github.com/test-kitchen/test-kitchen) from 3.3.1 to 3.3.2.
- [Release notes](https://github.com/test-kitchen/test-kitchen/releases)
- [Changelog](https://github.com/test-kitchen/test-kitchen/blob/main/CHANGELOG.md)
- [Commits](https://github.com/test-kitchen/test-kitchen/commits)

---
updated-dependencies:
- dependency-name: test-kitchen
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chef-ci committed:
Bump version to 5.21.4 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

dependabot[bot] committed:
Bump omnibus-software from `1d540dc` to `7d0e0fe` in /omnibus (#6274)

Bumps [omnibus-software](https://github.com/chef/omnibus-software) from `1d540dc` to `7d0e0fe`.
- [Release notes](https://github.com/chef/omnibus-software/releases)
- [Commits](https://github.com/chef/omnibus-software/compare/1d540dcdefae0fa75eb832590e85294e7c26660a...7d0e0fe472759f800a75fe65a1fe179da950c5c9)

---
updated-dependencies:
- dependency-name: omnibus-software
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chef-ci committed:
Bump version to 5.21.5 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
Fix for omnibus build failure on Windows (#6289)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.6 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

soumyo13 committed:
RESOURCE-527 Add an inspec init template for alicloud (#6304)

* adding the alicloud in the spec.files

Signed-off-by: Soumyodeep Karmakar <soumyo.k13@gmail.com>

* adding the test for generating inspec profile alicloud

Signed-off-by: Soumyodeep Karmakar <soumyo.k13@gmail.com>

* adding the alicloud profile

Signed-off-by: Soumyodeep Karmakar <soumyo.k13@gmail.com>

* fix lint error

Signed-off-by: Soumyodeep Karmakar <soumyo.k13@gmail.com>

Signed-off-by: Soumyodeep Karmakar <soumyo.k13@gmail.com>

chef-ci committed:
Bump version to 5.21.7 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

clintoncwolfe committed:
Fix Ruby 2.7 Bundle Installs on CI Verify Pipeline (#6324)

* We will never be on 2.6 again

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

* Only run html-proofer test on Ruby 3

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

* Pin to chef 16 on ruby 2.7, 17+ on Ruby 3

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

chef-ci committed:
Bump version to 5.21.8 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

sathish-progress committed:
CFINSPEC-493 update signing_identity (#6298)

* update signing_identity

Signed-off-by: Sathish <sbabu@progress.com>

* pin chef to older version for ruby version lesser than 3.0

Signed-off-by: Sathish <sbabu@progress.com>

Signed-off-by: Sathish <sbabu@progress.com>
Co-authored-by: Clinton Wolfe <clintoncwolfe@gmail.com>

chef-ci committed:
Bump version to 5.21.9 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

sathish-progress committed:
Cfinspec 488 fix nokigiri omnibus (#6305)

* update signing_identity

Signed-off-by: Sathish <sbabu@progress.com>

* pin chef 17.0 to ruby 2.7

Signed-off-by: Sathish <sbabu@progress.com>

* add nokogiri as dependency

Signed-off-by: Sathish <sbabu@progress.com>

* pin chef to older version for ruby version lesser than 3.0

Signed-off-by: Sathish <sbabu@progress.com>

* Linting

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

Signed-off-by: Sathish <sbabu@progress.com>
Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>
Co-authored-by: Clinton Wolfe <clintoncwolfe@gmail.com>

chef-ci committed:
Bump version to 5.21.10 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

clintoncwolfe committed:
Remove Windows 3.0 testing. Chef 18 was never released for Ruby 3.0, only 3.1 (#6330)

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

chef-ci committed:
Bump version to 5.21.11 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

dtheyer committed:
Prevent negative status from crashing launchd service resource (#6262)

Signed-off-by: dtheyer <davidheyer2016@gmail.com>

Signed-off-by: dtheyer <davidheyer2016@gmail.com>

chef-ci committed:
Bump version to 5.21.12 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

tnir committed:
Update chefstyle from 2.0.x to 2.2.2 to use RuboCop 1.25.1 (#6240)

Signed-off-by: Takuya Noguchi <takninnovationresearch@gmail.com>

Signed-off-by: Takuya Noguchi <takninnovationresearch@gmail.com>

chef-ci committed:
Bump version to 5.21.13 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

ahasunos committed:
Update readme for usage via Docker (CFINSPEC-516) (#6329)

* DOCS: Update readme for usage via Docker (CFINSPEC-516)

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* DOCS: Explain updated readme (CFINSPEC-516)

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

chef-ci committed:
Bump version to 5.21.14 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
CFINSPEC-393 - Fix for custom plugin loading issue when gemspec file is invalid and unable to load the lib path. (#6334)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.15 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
CFINSPEC-517: Fix for habitat buld failure  (#6340)

* Fix for habitat buld failure - Git load issue

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* Updating ruby installer version to 3.2.1

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* Revert "Updating ruby installer version to 3.2.1"

This reverts commit 53bae0f71e1e823f241ed1ba77209c04b11397fb.

* Checking removing the lines which are not needed.

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* Revert "Checking removing the lines which are not needed."

This reverts commit b5561abc515500f5738ceb84c1eb5c46760bc67f.

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.16 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Nik08 committed:
Upgraded mock ubuntu test from 20.04 to 22.04 (#6351)

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

magicmicah committed:
Fixing typo in user_permissions (#6344)

* Fixing typo in user_permissions

Signed-off-by: Magruder, Micah <Micah.Magruder@oracle.com>

* Corrected spacing

Signed-off-by: Magruder, Micah <Micah.Magruder@oracle.com>

Signed-off-by: Magruder, Micah <Micah.Magruder@oracle.com>

chef-ci committed:
Bump version to 5.21.17 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

clintoncwolfe committed:
Fix env smoke test by updating ERB.new in `inspec env`; add additional test (#6342)

* Failing test for Ruby 3.1 - ERB.new call options changed

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

* Fix for Ruby 3.1 by using named params to ERB.new

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

* Restrict test to non-windows systems

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

* Restrict test to non-windows systems

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

Vasu1105 committed:
CFINSPEC-533: Fix undefined method 'summary' for Gem::SourceFetchProblem (NoMethodError) when airgapped  (#6337)

* This fixes the undefined method 'summary' for #<Gem::SourceFetchProblem:0x000000000139dc70> (NoMethodError) while invoking any inspec command if the custom plugin is installed and if inspec commands are run in an air gap environment.
This fix removes an earlier call for getting gem specification from rubygems.org which is the cause for above error and replaces it with fetching the summary from a locally saved gemspec file. This will work in both airgap and non-airgap environments.

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* Adds test for reading description from local gemspec file.

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.18 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

clintoncwolfe committed:
Fix omnibus build which is picking ruby 2.7.4 which has vulnerability (#6341)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>
Co-authored-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.19 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

chef-ci committed:
Bump version to 5.21.20 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

chef-ci committed:
Update CHANGELOG.md with details from pull request #6341

Obvious fix; these changes are the result of automation not creative thinking.

Nik08 committed:
Revert "CFINSPEC-240 Extended file format support for waivers (#6193)" (#6366)

This reverts commit b7ddac9dcc4aa2bddfac940511d2fc47cae9b8c1.

chef-ci committed:
Bump version to 5.21.21 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
Revert the nokogiri dependency from omnibus project configuration (#6372)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.22 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
CFINSPEC-553: Fix failing verify pipeline builds on ruby 3.0 (#6378)

* bundle install without kitchen group from Gemfile as verify pipeline is breaking for ruby 3.0 as Chef 18 requires to be work with ruby 3.1. In future we will be removing the support for ruby 3.0 as Chef 18 never released with ruby 3.0

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* Removes all excluded groups from the bundle config set which was removed from the Gemfile.

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* Exclude the gem dependencies for kitchen for omnibus builds

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* Exclude the kitchen dependencies on windows platform too

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.23 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
Ruby 3.1 support: Adds ucrt platform to resolve windows omnibus build. (#6381)

* Adds the ucrt platform support for Windows

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* Adds the environment variable to ignore artifactory proxy as some gem are getting 500 error when tries to download from artifactory

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.24 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Nik08 committed:
CFINSPEC-240 Extended file format support for waivers: JSON & CSV (#6369)

* Revert "Revert "CFINSPEC-240 Extended file format support for waivers (#6193)" (#6366)"

This reverts commit 30146e07fe58426402a1f1ca358219ea582faca2.

* Waiver support for csv and json

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Removed the word excel from image tag

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Update waivers.md

Signed-off-by: Ian Maddaus <ian.maddaus@progress.com>

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>
Signed-off-by: Ian Maddaus <ian.maddaus@progress.com>
Co-authored-by: Ian Maddaus <ian.maddaus@progress.com>

chef-ci committed:
Bump version to 5.21.25 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Nik08 committed:
CFINSPEC-542 Bug fix for profiles with dependent profiles (#6377)

* Matching semver using regex to determine profile name without version in dsl logic

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Testing for selecting control using version in require control dsl

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Documentation for using version with profile name when including or selecting controls in a profile

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Commenting in dsl library about logic for fetching version and matching it

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Comment for regex used to fetch version

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Minor improvement for regex variable placement

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

chef-ci committed:
Bump version to 5.21.26 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
Bump version of omnibus-software (#6385)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.27 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
Add ruby-msys2-devkit as dependency and check (#6391)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

sean-sype-simmons committed:
adding new private verify, as this will kick off the expeditor bump version (#6397)

Signed-off-by: Sean Simmons <ssimmons@progress.com>

Signed-off-by: Sean Simmons <ssimmons@progress.com>

chef-ci committed:
Bump version to 5.21.28 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

clintoncwolfe committed:
Update copyright year in README (#6395)

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

chef-ci committed:
Bump version to 5.21.29 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

chef-ci committed:
Executed '.expeditor/update_dockerfile.sh'

Obvious fix; these changes are the result of automation not creative thinking.

ahasunos committed:
CFINSPEC-522: Remove rake tasks which are no longer used (#6367)

* CLEAN-UP: Remove unused rake tasks
missing, kitchen and version tasks are not used

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* CLEAN-UP: Remove functional rake test
test:default covers both unit and functional tests
default was introduced to cover both functional and unit in PR #4102

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* CLEAN-UP: Remove unused docs related tasks

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* CLEAN-UP: Remove tasks/* from expeditor config

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* REVERT: Undo deletion of functional test

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* RECOVER: Retain docs:cli task

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* UNDO: Revert changes to config

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

* CHORE: Add reference to this PR in note for future use

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

---------

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

ahasunos committed:
RAKE TEST: Fix rake task for docs:cli

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

ahasunos committed:
DOCS: Generate cli.md with rake docs:cli

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

ahasunos committed:
Remove unused require

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

ahasunos committed:
FORMAT: Use html dl, dt, dd tags for displaying subcommands

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

ahasunos committed:
REFACTOR: Remove repetitive dl closing for subcommands of single command

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

ahasunos committed:
Merge pull request #6384 from inspec/ss/update-cli-docs-rake-task

CFINSPEC-554: Update CLI Docs rake task to work with current InSpec docs.

chef-ci committed:
Bump version to 5.21.30 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
CFINSPEC-573: Fix for mongo_session resources prints debug level of information in output of the run result. (#6401)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

Vasu1105 committed:
Updates release process docs as per current changes (#6398)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.31 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

chef-ci committed:
Bump version to 5.21.32 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
CFINSPEC-576 : Fix profile gem dependency loading issue when dependent gem is required inside profile libraries. (#6408)

* Test profile consist of custom resource which has a gem dependency

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* CFINSPEC-576 Fix for profile gem depedency loading issue if depedent gem is required in libraries files of the profile

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* Fix windows test failure for profile gem dependency

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

---------

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.33 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Nik08 committed:
CFINSPEC-580 Dependent profiles fix for any scheme of version used in profiles (#6410)

* Dependent profiles fix for any scheme of version used in profiles

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Dependent profile fix when version is blank in dependent profiles

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

---------

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

chef-ci committed:
Bump version to 5.21.34 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

IanMadd committed:
Formatting fixes for podman docs (#6386)

* Formatting fixes for podman docs

Signed-off-by: Ian Maddaus <ian.maddaus@progress.com>

* Minor fix

Signed-off-by: Ian Maddaus <ian.maddaus@progress.com>

* Update docs-chef-io/content/inspec/resources/podman.md

Signed-off-by: Ian Maddaus <ian.maddaus@progress.com>

Co-authored-by: Vasundhara Jagdale <vasundhara.jagdale@chef.io>

* Update docs-chef-io/content/inspec/resources/podman.md

Signed-off-by: Ian Maddaus <ian.maddaus@progress.com>

Co-authored-by: Vasundhara Jagdale <vasundhara.jagdale@chef.io>

---------

Signed-off-by: Ian Maddaus <ian.maddaus@progress.com>
Co-authored-by: Vasundhara Jagdale <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.35 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
CFINSPEC-550 Add RHEL9 to build matrix (#6403)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

dependabot[bot] committed:
Bump omnibus-software from `6a13693` to `225e357` in /omnibus (#6442)

Bumps [omnibus-software](https://github.com/chef/omnibus-software) from `6a13693` to `225e357`.
- [Release notes](https://github.com/chef/omnibus-software/releases)
- [Commits](https://github.com/chef/omnibus-software/compare/6a13693685cc7ab4f8d3592c80ad612cc8c6f1ac...225e3576c48fcd0155f6049cb032b2370eccf29a)

---
updated-dependencies:
- dependency-name: omnibus-software
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chef-ci committed:
Bump version to 5.21.36 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

dependabot[bot] committed:
Bump omnibus from `c66e97c` to `cf97613` in /omnibus (#6423)

Bumps [omnibus](https://github.com/chef/omnibus) from `c66e97c` to `cf97613`.
- [Release notes](https://github.com/chef/omnibus/releases)
- [Commits](https://github.com/chef/omnibus/compare/c66e97c211a60296e58da100d79d048974e13904...cf9761311577e24819625aa1d932f828740e33b4)

---
updated-dependencies:
- dependency-name: omnibus
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chef-ci committed:
Bump version to 5.21.37 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

dependabot[bot] committed:
Bump kitchen-vagrant from 1.12.1 to 1.14.1 in /omnibus (#6427)

Bumps [kitchen-vagrant](https://github.com/opscode/kitchen-vagrant) from 1.12.1 to 1.14.1.
- [Release notes](https://github.com/opscode/kitchen-vagrant/releases)
- [Changelog](https://github.com/test-kitchen/kitchen-vagrant/blob/main/CHANGELOG.md)
- [Commits](https://github.com/opscode/kitchen-vagrant/compare/v1.12.1...v1.14.1)

---
updated-dependencies:
- dependency-name: kitchen-vagrant
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chef-ci committed:
Bump version to 5.21.38 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

dependabot[bot] committed:
Bump test-kitchen from 3.3.2 to 3.5.0 in /omnibus (#6360)

Bumps [test-kitchen](https://github.com/test-kitchen/test-kitchen) from 3.3.2 to 3.5.0.
- [Release notes](https://github.com/test-kitchen/test-kitchen/releases)
- [Changelog](https://github.com/test-kitchen/test-kitchen/blob/main/CHANGELOG.md)
- [Commits](https://github.com/test-kitchen/test-kitchen/commits/v3.5.0)

---
updated-dependencies:
- dependency-name: test-kitchen
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chef-ci committed:
Bump version to 5.21.39 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

dependabot[bot] committed:
Bump rack from 2.2.4 to 2.2.6.2 in /omnibus (#6388)

Bumps [rack](https://github.com/rack/rack) from 2.2.4 to 2.2.6.2.
- [Release notes](https://github.com/rack/rack/releases)
- [Changelog](https://github.com/rack/rack/blob/main/CHANGELOG.md)
- [Commits](https://github.com/rack/rack/compare/2.2.4...v2.2.6.2)

---
updated-dependencies:
- dependency-name: rack
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chef-ci committed:
Bump version to 5.21.40 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
Revert "adding new private verify, as this will kick off the expeditor bump version (#6397)" (#6479)

This reverts commit 227b0f2d3577a9c2b13cc6a87cfe9bc81e49662a.

chef-ci committed:
Bump version to 5.21.41 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Nik08 committed:
CHEF-1458 Multiple values changes in SimpleConfig library (#6489)

* Simple config changes to return array of values with multiple values opt

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* New option added to split multiple values with a delimiter regex

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

---------

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

chef-ci committed:
Bump version to 5.21.42 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
Adds configuration required for sonarqube integration. (#6480)

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.21.43 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

jjacque committed:
Add nftables resources (#6499)

* Fix systemd path for Leap image

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Use vhef client version 17 as doocker cookbook do not support >= 18

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Add nftables resource

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Add nftables tests

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Add fixtures for nftables tests

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* enable nftables only when attr is true - then disable iptables

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* By default test iptables, not nftables

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Fix tests and lint errors

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Increase unit test coverage for nftables

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Do not use -nn nft option as behaviour changes based on nft version

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Base nft params identification on its version, not os version

    Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Make test more human friendly by reversing unless/if logic

    Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Update mocked cmds with nft params

    Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Fix quoting issue with rubocop

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Fix uninitiallized class vars

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Fix unit test by adding nft version mocking

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

* Clean nftables doc

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

---------

Signed-off-by: Jeremy JACQUE <jeremy.jacque@algolia.com>

chef-ci committed:
Bump version to 5.21.44 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

clintoncwolfe committed:
Clarify key_rsa docs regarding SSH keys (#6507)

* Clarify that an RSA resource can only work on RSA keypairs

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

* Update docs-chef-io/content/inspec/resources/key_rsa.md

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

Co-authored-by: Ian Maddaus <IanMadd@users.noreply.github.com>

---------

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>
Co-authored-by: Ian Maddaus <IanMadd@users.noreply.github.com>

clintoncwolfe committed:
CHEF-2438 Add train-kubernetes to inspec gemspec (#6512)

* Add train-kubernetes to inspec gemspec, fix spacing

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

* Fix failing test to detect system plugins

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

---------

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>
Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>
Co-authored-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.22.0 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

dependabot[bot] committed:
Bump omnibus from `cf97613` to `ae855f3` in /omnibus (#6515)

Bumps [omnibus](https://github.com/chef/omnibus) from `cf97613` to `ae855f3`.
- [Release notes](https://github.com/chef/omnibus/releases)
- [Commits](https://github.com/chef/omnibus/compare/cf9761311577e24819625aa1d932f828740e33b4...ae855f306015f1779684f490932a633055ececa6)

---
updated-dependencies:
- dependency-name: omnibus
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump rack from 2.2.6.2 to 2.2.6.4 in /omnibus (#6490)

Bumps [rack](https://github.com/rack/rack) from 2.2.6.2 to 2.2.6.4.
- [Release notes](https://github.com/rack/rack/releases)
- [Changelog](https://github.com/rack/rack/blob/main/CHANGELOG.md)
- [Commits](https://github.com/rack/rack/compare/v2.2.6.2...v2.2.6.4)

---
updated-dependencies:
- dependency-name: rack
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chef-ci committed:
Bump version to 5.22.1 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

dependabot[bot] committed:
Bump omnibus-software from `225e357` to `f06bbea` in /omnibus (#6508)

Bumps [omnibus-software](https://github.com/chef/omnibus-software) from `225e357` to `f06bbea`.
- [Release notes](https://github.com/chef/omnibus-software/releases)
- [Commits](https://github.com/chef/omnibus-software/compare/225e3576c48fcd0155f6049cb032b2370eccf29a...f06bbea744458f2cb63c98e6891f9cfb904a1f90)

---
updated-dependencies:
- dependency-name: omnibus-software
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

amlodzianowski committed:
fix: ensure Invoke-WebRequest headers can be configured (#6484)

* fix: ensure Invoke-WebRequest headers can be configured

Signed-off-by: Adrian Mlodzianowski <amlodzianowski@logicworks.net>

* fix: use array for custom headers

Signed-off-by: Adrian Mlodzianowski <amlodzianowski@logicworks.net>

* test: add unit test for Windows simple HTTP request with headers

Signed-off-by: Adrian Mlodzianowski <amlodzianowski@logicworks.net>

---------

Signed-off-by: Adrian Mlodzianowski <amlodzianowski@logicworks.net>
Co-authored-by: Adrian Mlodzianowski <amlodzianowski@logicworks.net>

chef-ci committed:
Bump version to 5.22.2 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Taknok committed:
For #6493 : Add postgres_session support for custom port with a socket connection (#6494)

* Add support for custom port with a socket connection

Signed-off-by: Pg <pg.developper.fr@gmail.com>

* Add tests for the postgres_session with custom port

Signed-off-by: Pg <pg.developper.fr@gmail.com>

---------

Signed-off-by: Pg <pg.developper.fr@gmail.com>

chef-ci committed:
Bump version to 5.22.3 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

AlanFoster committed:
Update host resource to resolve all ipaddresses (#6481)

Signed-off-by: Alan Foster <alan@alanfoster.me>

clintoncwolfe committed:
CHEF-1631 Clarify that command timeout default was withdrawn (#6511)

* Clarify that command timeout default was withdrawn

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

* Delete reference to timeout default entirely.

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

Co-authored-by: Ian Maddaus <IanMadd@users.noreply.github.com>

---------

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>
Co-authored-by: Ian Maddaus <IanMadd@users.noreply.github.com>

chef-ci committed:
Bump version to 5.22.4 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

chef-ci committed:
Executed '.expeditor/update_dockerfile.sh'

Obvious fix; these changes are the result of automation not creative thinking.

pirj committed:
Update rspec to 3.12 (#6523)

Update RSpec dependency. There are some fixes and no breaking changes in 3.12

Signed-off-by: Phil Pirozhkov <hello@fili.pp.ru>

chef-ci committed:
Bump version to 5.22.5 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

clintoncwolfe committed:
Update Docker base image to be ubuntu 22.04 (#6526)

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

chef-ci committed:
Update CHANGELOG.md with details from pull request #6526

Obvious fix; these changes are the result of automation not creative thinking.

ahasunos committed:
SUPPORT: Update dockerfile for m1 support (#6541)

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

chef-ci committed:
Update CHANGELOG.md with details from pull request #6541

Obvious fix; these changes are the result of automation not creative thinking.

dependabot[bot] committed:
Bump omnibus from `ae855f3` to `9c0643a` in /omnibus (#6543)

Bumps [omnibus](https://github.com/chef/omnibus) from `ae855f3` to `9c0643a`.
- [Release notes](https://github.com/chef/omnibus/releases)
- [Commits](https://github.com/chef/omnibus/compare/ae855f306015f1779684f490932a633055ececa6...9c0643a3a44f3e7119789c2093bfc8edd78c74ff)

---
updated-dependencies:
- dependency-name: omnibus
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chef-ci committed:
Bump version to 5.22.6 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

dependabot[bot] committed:
Bump berkshelf from 8.0.2 to 8.0.7 in /omnibus (#6530)

Bumps [berkshelf](https://github.com/chef/berkshelf) from 8.0.2 to 8.0.7.
- [Changelog](https://github.com/chef/berkshelf/blob/main/CHANGELOG.md)
- [Commits](https://github.com/chef/berkshelf/compare/v8.0.2...v8.0.7)

---
updated-dependencies:
- dependency-name: berkshelf
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chef-ci committed:
Bump version to 5.22.7 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

dependabot[bot] committed:
Bump omnibus-software from `f06bbea` to `4b08f0b` in /omnibus (#6548)

Bumps [omnibus-software](https://github.com/chef/omnibus-software) from `f06bbea` to `4b08f0b`.
- [Commits](https://github.com/chef/omnibus-software/compare/f06bbea744458f2cb63c98e6891f9cfb904a1f90...4b08f0bc0688f750bc55a49b8103b2d12815399e)

---
updated-dependencies:
- dependency-name: omnibus-software
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chef-ci committed:
Bump version to 5.22.8 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

sathish-progress committed:
Merge branch 'main' of github.com:inspec/inspec into inspec-6

clintoncwolfe commented on CHANGELOG.md:
```diff
@@ -6,10 +6,9 @@
 
```
Exclude the CHANGELOG.md from any conflict resolution attempt - I fully expect there to be conflicts when we merge the changelog, as we pulled the changelog from inspec-prime. 

It's also causing the PR to fail expeditor vaildation checks.

clintoncwolfe replied:
In discussion, Sathish explained these were missed changes that will eventually be resolved -might as well pull them in now. Works for me.

Nik08 committed:
YAML reporter test fix (#6563)

* Fix for yaml reporter data, not matching empty values with spaces

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Additional comment to document affecting fields

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

---------

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

chef-ci committed:
Bump version to 5.22.9 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Nik08 committed:
Merged main after yaml reporter test fix

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

clintoncwolfe committed:
Add inspec-6 branch to Dependabot monitoring. This change must be made on main for Dependabot to pick it up. (#6569)

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

chef-ci committed:
Update CHANGELOG.md with details from pull request #6569

Obvious fix; these changes are the result of automation not creative thinking.

Nik08 committed:
CHEF-4115 Added ability to merge reporter configurations from both CLI and config (#6568)

* Added ability to merge cli and config reporter options

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Test cases to validate working of reporter configuration using cli and config

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Documentation change to add information on reporter configurations usage with both

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Added Doc review changes and text fixture for config json

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Verify fix

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Verify pipeline test fixes for reporter options to be read successfully

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Test changes in both cli and config reporter usage scenarios to fix verify pipeline

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Review comments to improvise

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

* Renamed testing fixture file for reporter cli config file

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

---------

Signed-off-by: Nikita Mathur <nikita.mathur@chef.io>

chef-ci committed:
Bump version to 5.22.10 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

ahasunos committed:
EULA: Update installer license text (#6582)

Signed-off-by: Sonu Saha <sonu.saha@progress.com>

chef-ci committed:
Bump version to 5.22.11 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

Vasu1105 committed:
CHEF-5200 Waived controls are not getting waived (skipped) in case of failure at resource level. (#6588)

* CHEF-5200: This fix applies waiver if exception occurs while eval the control inside control block. This ensures that waiver are got applied in case of resouce level failures too.

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* When waiver file is empty error should be raised the written is invalid for the empty waiver file.
Added different test to check only_if waiver is applied when the waived control in the waived file has expired.

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* Fix lint

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

* Fix test failing on windows due to different exit code on the Windows.

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

---------

Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

chef-ci committed:
Bump version to 5.22.12 by Chef Expeditor

Obvious fix; these changes are the result of automation not creative thinking.

sathish-progress committed:
remove kitchen group

sathish-progress committed:
Merge branch 'main' of github.com:inspec/inspec into missing-changes-from-main

Signed-off-by: Sathish <sbabu@progress.com>

sathish-progress committed:
Merge branch 'inspec-6' of github.com:inspec/inspec into missing-changes-from-main

Signed-off-by: Sathish <sbabu@progress.com>

sathish-progress committed:
keep waiver refactoring for inspec-6

Signed-off-by: Sathish <sbabu@progress.com>
